app/models/group.rb

```diff
@@ -1,21 +1,37 @@
-class Group < ApplicationRecord
-  acts_as_tenant(:tenant)
+class Group
+  attr_accessor :name
+  attr_accessor :description
+  attr_accessor :uuid
+  attr_writer   :users
 
-  validates :name, :presence => true
+  def self.find(uuid)
```

Comment:
@bzwei I do agree with @mkanoor in regards to the question that these classes should be services since they are no longer AR backed. 
I also realize these changes to work with RBAC are extensive. I think they ultimately should be moved but as a follow up when we have time for such things

Reply:
Agreed, this is a great first step in switching between local user/groups and moving to the RBAC service.  We do not have to make the leap in one PR.

@bzwei I would suggest opening a git issue which we can link to this PR to track that effort.

Reply:
https://github.com/ManageIQ/approval-api/issues/57